cafebabel/static/js/edit-article.js

```diff
@@ -1,36 +1,48 @@
-const tagButtonAdd = document.querySelector('.tags button.add')
-const tagButtonRemove = document.querySelector('.tags button.remove')
-const tagsList = document.querySelector('.tags .tagsList')
+class Tags {
+  constructor(domContext) {
+    this.list = domContext.querySelector('ul')
+    this.values = Array.from(this.list.querySelectorAll('li')).map(li => li.innerText)
+    this.submission = domContext.querySelector('input[name=tag-new]').value
+  }
+  checkSubmission() {
+    return this.submission && ! this._isTag(this.submission)
+  }
+  display() {
```

Comment:
and here

Reply:
I'd suggest calling it `render` as it seems to become more of a convention.
But this is subjective, up to you ;)